conjur/api.py

```diff
@@ -126,7 +126,7 @@ def authenticate(self):
         return invoke_endpoint(HttpVerb.POST, ConjurEndpoint.AUTHENTICATE, params,
                                self.api_key, ssl_verify=self._ssl_verify).text
 
-    def resources_list(self, list_constraints):
+    def resources_list(self, *list_constraints):
```

Comment:
maybe we can just call it args and not list_constraints?

Reply:
they will always be list constraints. list_constraints represents the flags which put constraints on the response